flatdata-py/generator/tests/generators/schemas.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+def schemas_and_expectations(generator, extension):
```

Comment:
Could we add a little bit of documentation what are `generator` and `extension` and what is the return type. Unfortunately, it is hard to read python without static types.

Reply:
Done